File: app/src/main/java/com/nollpointer/dates/views/SortCardsControl.java
package com.nollpointer.dates.views;

import android.content.res.Resources;

import java.util.List;

public interface SortCardsControl {

    void setQuestions(List<String> list);

    void setAnswerSequence(int[] sequence);

    boolean check();

    void setCheckMode(boolean state);

    void setColors(int g, int r);
}

File: app/src/main/java/com/nollpointer/dates/sort/SortCardsControl.kt
package com.nollpointer.dates.sort

interface SortCardsControl {
    fun setQuestions(list: List<String?>?)
    fun setAnswerSequence(sequence: IntArray?)
    fun check(): Boolean
    fun setCheckMode(state: Boolean)
    fun setColors(g: Int, r: Int)
}